### PR TITLE
Uses unicode_literals from future

### PR DESCRIPTION
**Pull request by sopelj**
reason: Future python 3 compatibility

### CODE REVIEW

File: wazo_confgend/generators/tests/test_extensionsconf.py [hunk 1]
```diff
@@ -144,7 +146,7 @@ def test_generate(self, mock_asterisk_conf_dao, mock_ivr_dao):
 
         mock_ivr_dao.find_all_by.return_value = [
             IVR(id=42, name='foo', menu_sound='hello-world'),
-            IVR(id=43, name='bar', menu_sound='youhou'),
+            IVR(id=43, name='bar', menu_sound='youho'),
```
Comment (fblackburn1) [inline wazo_confgend/generators/tests/test_extensionsconf.py:149]: Out of curiosity, why changing this string? :smile:

Comment (sduthil) [inline wazo_confgend/generators/tests/test_extensionsconf.py]: Errr... Was that on purpose? :sweat_smile:

**Review by sduthil — CHANGES_REQUESTED** [review]
Oh thank you so much for this!!!

Just spotted one word that should not have been changed, but did not find any other error mistake :)

File: wazo_confgend/generators/tests/test_extensionsconf.py (hunk 1 above)
Comment (sduthil) [reply wazo_confgend/generators/tests/test_extensionsconf.py]: After checking, this value is not really useful in the test... So I'll make the change (I have it right in front of me).

Comment (sduthil) [reply wazo_confgend/generators/tests/test_extensionsconf.py]: Sorry that is already fixed. Merging...

**Review by sduthil — APPROVED** [review]

File: wazo_confgend/generators/tests/test_extensionsconf.py (hunk 1 above)
Comment (sopelj) [reply wazo_confgend/generators/tests/test_extensionsconf.py]: Haha, no it wasn't on purpose, but yeah, it was fixed in the following commit. Good catch though! Thanks! :)

**Pull request merged**